Add Hex Code

Pixelatedza opened the pull request:
Adding Hexadecimal Code of current character.

Pixelatedza committed:
Add Hex Code

Adding Hexadecimal Code of current character.

borislubimov commented on show_char_code.py:
```diff
@@ -11,4 +11,4 @@ def on_activated(self, view):
 
     def show_char_code(self, view):
         selected = view.substr(view.sel()[0].a)
-        view.set_status('charcode', "Char Code: {0}".format(ord(selected)))
```
Hi, thanks for PR! I think it's better to use `{0:#x}` to display hex code. Also I noticed that in [fork of my repo](https://github.com/onelittlehope/ShowCharacterCode/blob/master/show_char_code.py) already added hex and byte codes. Maybe it's worth to do here the same so these changes would be available through Package Control. The only thing I'm not sure is the name "BYTE". Is that name correct for such character representation?

Pixelatedza replied:
I liked the cleaner look, but I agree that #x is a better approach.

I saw the fork doing something similar. I didn't want to clutter up the bottom too much and thought binary might be a bit excessive. Maybe making it an option that can be turned on/off?

Pixelatedza committed:
Added Binary

Added binary of the current character.

Changed Hex representation to more formal notation. Ex: 15 would now
display 0x15.

Pixelatedza commented:
I've added the code for binary representation as well.

borislubimov commented:
Binary representation indeed a bit excessive, so I replaced it with byte representation which is more practical
